n26/api.py

```diff
@@ -452,7 +452,7 @@ def _complete_authentication_flow(self, mfa_token: str) -> dict:
             mfa_response_data['grant_type'] = "mfa_otp"
 
             hint = click.style("Enter the 6 digit SMS OTP code", fg="yellow")
-            mfa_response_data['otp'] = click.prompt(hint, type=int)
+            mfa_response_data['otp'] = click.prompt(hint)
```

Comment:
I'd prefer an explicit `type=str` with a comment above that explains it. This way we will know why this is important lateron.

Reply:
There!